### ISSUE TITLE
Accessibility: Mermaid diagrams frequently violate WCAG colour contrast requirements

### ISSUE THREAD

**Issue by MaddyGuthridge** [microsoft/vscode#328924]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.130 (I delay updates intentionally for stability reasons, but have reproduced on newer versions too)
- OS Version: NixOS 26.05 Yarara

Steps to Reproduce:

1. Open a Markdown document
2. Create a Mermaid diagram

I have found that many aspects of Mermaid diagrams are inaccessible to the point of unreadability (even for me, someone without vision impairments), across many built-in themes. I am unable to read mermaid diagrams in VS Code as a consequence of this poor accessibility.

For example:

````md
```mermaid
sequenceDiagram
    Alice->>+John: Hello John, how are you?
    Alice->>+John: John, can you hear me?
    John-->>-Alice: Hi Alice, I can hear you!
    John-->>-Alice: I feel great!
```
````

### Theme "Dark 2026"

<img width="488" height="408" alt="Image" src="https://github.com/user-attachments/assets/62839aa3-dcac-4e0f-80c1-626504bbd6a8" />

For the seuqnece diagram activation bars, WebAIM reports a contrast ratio of 1.21:1.

### Theme "Quiet Light"

<img width="488" height="408" alt="Image" src="https://github.com/user-attachments/assets/b34f35f1-6928-4c76-9998-0f2498847904" />

Text has a contrast ratio of 1.09:1.

### Theme "Light 2026"

<img width="488" height="408" alt="Image" src="https://github.com/user-attachments/assets/1faf8a43-c052-4909-8db6-80b11caa01c1" />

The text is white-on-white.

**Comment by MaddyGuthridge** [comment]
Issue also exists on latest.

**Comment by MaddyGuthridge** [comment]
Hello! I am bumping this issue again. I tutor a course where we expect students to write mermaid diagrams using VS Code, and the accessibility violations are a major concern for ensuring the course is accessible and equitable. Please can this issue be prioritised for an early fix?
